tracer/src/Datadog.Trace/Vendors/dnlib/DotNet/ModuleDefMD.cs

```diff
@@ -1499,7 +1499,9 @@ internal sealed class ModuleDefMD : ModuleDefMD2, IInstructionOperandResolver {
 			return new EmbeddedResourceMD(this, mr, Array2.Empty<byte>());
 		}
 
+#pragma warning disable SYSLIB0032 // HandleProcessCorruptedStateExceptionsAttribute' is obsolete: 'Recovery from corrupted process state exceptions is not supported; HandleProcessCorruptedStateExceptionsAttribute is ignored'
```

Comment:
NIT: Maybe we should use the GlobalSuppressions file to globally suppress the warning on `Vendors/dnlib`. This pragma will be lost if we update the vendored code.

Reply:
Yeah, I wondered that, but IIRC, @OmerRaviv had to do a lot of work to get it imported, so figured it wasn't a big deal. I might do in a tidying up PR, just to save the CI 😬

Reply:
Oy, tbh I hadn't noticed all these warnings 'dnlib' introduced. Lemme follow up next week with a PR to clean these up (just noticed there are a lot of'em).